parm/soca/obs/config/adt_j3.yaml

```diff
@@ -1,9 +1,9 @@
 obs space:
   name: adt_j3
   obsdatain:
-    obsfile: $(experiment_dir)/{{current_cycle}}/adt_j3.{{window_begin}}.nc4
+    obsfile: ./obs/adt_j3.{{window_begin}}.nc4
```

Comment:
Inconsistency between `./obs` and `obs_out`, both are probably the same but should be consistent with path formats